functional-tests/fn_prover_cl_dispatch.py

```diff
@@ -13,17 +19,27 @@ def __init__(self, ctx: flexitest.InitContext):
 
     def main(self, ctx: flexitest.RunContext):
         prover_client = ctx.get_service("prover_client")
+        seq = ctx.get_service("sequencer")
+
         prover_client_rpc = prover_client.create_rpc()
+        seqrpc = seq.create_rpc()
 
         # Wait for the Prover Manager setup
         time.sleep(5)
 
         # Dispatch the prover task
-        task_ids = prover_client_rpc.dev_strata_proveCLBlock(1)
-        self.debug(f"got task ids: {task_ids}")
+        start_block_id = self.blockidx_2_blockid(seqrpc, CL_PROVER_PARAMS["start_block"])
+        end_block_id = self.blockidx_2_blockid(seqrpc, CL_PROVER_PARAMS["end_block"])
+
+        task_ids = prover_client_rpc.dev_strata_proveClBlocks((start_block_id, end_block_id))
         task_id = task_ids[0]
+
         self.debug(f"using task id: {task_id}")
         assert task_id is not None
 
         time_out = 10 * 60
         wait_for_proof_with_time_out(prover_client_rpc, task_id, time_out=time_out)
+
+    def blockidx_2_blockid(self, seqrpc, blockidx):
+        l2_blks = seqrpc.strata_getHeadersAtIdx(blockidx)
+        return l2_blks[0]["block_id"]
```

Comment:
Have been trying to use the term "slot" more often since it's more clear than blockidx and less prone to accidental misreading/mistypes.

Since this doesn't reference `self`, should we move it out of the class to a toplevel fn or even into `utils`?